packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -123,6 +128,70 @@ class EmbeddedContentConsumer extends React.Component {
     this.addEventListeners(this.props.eventHandlers);
   }
 
+  handleFrameVisualFocusIndicator() {
+    // Selectors for interactable elements
+    const interactableElementSelector = 'a[href]:not([tabindex=\'-1\']), area[href]:not([tabindex=\'-1\']), input:not([disabled]):not([tabindex=\'-1\']), '
+      + "select:not([disabled]):not([tabindex='-1']), textarea:not([disabled]):not([tabindex='-1']), button:not([disabled]):not([tabindex='-1']), "
+      + "[contentEditable=true]:not([tabindex='-1'])";
+
+    // Check if the content is scrollable
+    const isContentScrollable = () => {
+      const doc = this.xfcFrame?.iframe?.contentWindow?.document;
+      return (doc.documentElement.scrollHeight > doc.documentElement.clientHeight
+        || doc.body.scrollHeight > doc.body.clientHeight
+        || doc.documentElement.scrollWidth > doc.documentElement.clientWidth
+        || doc.body.scrollWidth > doc.body.clientWidth);
```

Comment:
It would probably be helpful to add a comment to explain this logic.  It looks like you are checking both the HTML element and body elements for scrolling.

Reply:
https://github.com/cerner/terra-framework/pull/1874/files#diff-77b1c1b39ad95249bd3956ef509a0995b4687f8c94ba87a3110f3e6b1ce8ccfeR136-R148